packages/blocks/src/store/process-block-type.js

```diff
@@ -62,6 +62,34 @@ function mergeBlockVariations(
 	return result;
 }
 
+/**
+ * Function to stabilize border supports.
+ * This function needs to be merged once we have below PR merged.
+ *
+ *  Reference from - https://github.com/WordPress/gutenberg/pull/63401/
+ *
+ *
+ * @param {Object} rawSupports Support for the block.
+ *
+ * @return {Object} Stabilized supports.
+ */
+function stabilizeBorderSupports( rawSupports ) {
+	if ( ! rawSupports ) {
+		return rawSupports;
+	}
+
+	const supports = { ...rawSupports };
+
+	if (
+		supports?.__experimentalBorder &&
+		typeof supports.__experimentalBorder === 'object'
+	) {
+		supports.border = supports.__experimentalBorder;
```

Comment:
This needs to merge with values already using the stabilized `border` property rather than flat out replacing it.